src/Builders/BillingItemBuilder.php

```
@@ -2,63 +2,220 @@
+	 * @param int   $pageSize
+	 * @param array $filters
+	 *
+	 * @return \Illuminate\Support\Collection|ExternalBillingItem[]
+	 * @throws \LasseRafn\Economic\Exceptions\EconomicClientException
+	 * @throws \LasseRafn\Economic\Exceptions\EconomicRequestException
+	 */
+	public function getByPage($page = 0, $pageSize = 500, $filters = [])
+	{
+		$items = collect([]);
+
+		$urlQuery = \LasseRafn\Economic\Services\QueryGeneratorService::generateQuery($filters, [], true);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\LasseRafn\Economic\Services\QueryGeneratorService' in method 'getByPage'. (undefined)

(StaticAccess)